<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QField</class>
 <widget class="QWidget" name="QField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configure Project for QField Synchronisation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="exportTypeTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="cloudExportTab">
      <attribute name="title">
       <string>QFieldCloud</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,0">
       <item row="0" column="0">
        <widget class="QRadioButton" name="preferOnlineLayersRadioButton">
         <property name="text">
          <string>Prefer online layers</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="preferOfflineLayersRadioButton">
         <property name="text">
          <string>Prefer offline layers</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QgsCollapsibleGroupBox" name="cloudAdvancedSettings">
         <property name="title">
          <string>Advanced Settings</string>
         </property>
         <property name="collapsed">
          <bool>true</bool>
         </property>
         <property name="saveCheckedState">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cableExportTab">
      <attribute name="title">
       <string>Cable Export</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="createBaseMapGroupBox">
     <property name="title">
      <string>Base Map</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="saveCheckedState">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="singleLayerRadioButton">
        <property name="text">
         <string>Single layer</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QStackedWidget" name="baseMapTypeStack">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDialableCharactersOnly</set>
        </property>
        <widget class="QWidget" name="mapThemePage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Map Theme</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mapThemeComboBox"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="singleLayerPage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Layer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsMapLayerComboBox" name="layerComboBox"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="tileSize">
        <property name="toolTip">
         <string>Rendering will happen in tiles. This number determines the width and height (in pixels) that will be rendered per tile.</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="text">
         <string>1024</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mapUnitsPerPixel">
        <property name="toolTip">
         <string>This determines the spatial resolution of the resulting map image. It depends on the CRS of the map canvas. For map units in [m], a value of 1 means each pixel covers an area of 1x1 m, a value of 1000 means 1 pixel per square kilometer.</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mapUnitsPerPixelLabel">
        <property name="text">
         <string>Map Units per Pixel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="tileSizeLabel">
        <property name="text">
         <string>Tile Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="mapThemeRadioButton">
        <property name="text">
         <string>Map theme</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Advanced Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="onlyOfflineCopyFeaturesInAoi">
        <property name="text">
         <string>Only copy features in area of interest</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
